.github/drafts/16506/16506_number_slider_outline.md

```diff
@@ -0,0 +1,67 @@
+*** Range Slider ***
+
+The Number Slider widget allows users to choose a numerical value from a range of data.
+
+< TODO: NEED VIDEO DEMO >
+
+---
+# Usage
+- Example use case: Page turner/selector for a multi-page document or content
+
+---
+
+# Properties
+
+Properties allow you to edit the widget, connect it with other widgets and customize the user actions.
+
+## Widget Properties
```

Comment:
Same as above - Please update based on the [widget template](https://www.notion.so/appsmith/Widget-Doc-template-5d7fa61a2e2a4ee888487ba9b43bf487#6df273cbb4e84edb94931b6e0bc59de1).